app/marketing/management/commands/assemble_leaderboards.py

```diff
@@ -33,6 +33,7 @@
 quarterly_cutoff = timezone.now() - timezone.timedelta(days=90)
 yearly_cutoff = timezone.now() - timezone.timedelta(days=365)
 
+
```

Comment:
@owocki  extra line added ?

Reply:
i think 2 line breaks is the standard before top level methods in pep8